.github/ISSUE_TEMPLATE/general_issue.md

```diff
@@ -0,0 +1,22 @@
+---
+name: General Issue
+about: Create a quick issue with minimal information
+title: ""
+assignees: ""
+---
+
+## Summary
+
+<!-- A brief description of the issue. -->
+
+## Details
+
+<!-- Any relevant details about the issue. -->
+
+## Action Items
```

Comment:
A bit vague. Is this for internal issues? Community issue creator will probably get confused with this.

Either remove it or add explicit instructions that it is for internal use.